src/Components/App/App.css

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
```

Comment:
I've seen people using this line of code in the past, but I don't know what it does....

I'm going to google it!